serde_v8/de.rs

```diff
@@ -615,3 +637,22 @@ impl<'de, 'a, 'b, 's> de::VariantAccess<'de>
     de::Deserializer::deserialize_struct(&mut d, "", fields, visitor)
   }
 }
+
+fn bigint_to_f64(b: v8::Local<v8::BigInt>) -> f64 {
```

Comment:
This seems questionable. Can a JS number not already represent all values in an `f64`?

Reply:
Yep, JS numbers are effectively doubles/f64s (sometimes SMIs) but this makes sense if we're being "permissive" in the inputs and want to allow deserializing a BigInt to an f64

Reply:
I don't think we want to allow this though. It seems that in 99% of cases this will hide a user mistake. People don't use BigInt to store f64s.

Reply:
> I don't think we want to allow this though. It seems that in 99% of cases this will hide a user mistake. People don't use BigInt to store f64s.

I don't disagree, it links back to the broader question if we want serde_v8 to be very strict or permissive, should it coerce implicitly or error instead.

Usually for the op-layer stricter is better